libcodechecker/analyze/analyzers/result_handler_plist_to_db.py

```diff
@@ -185,38 +185,18 @@ def __store_bugs(self, files, reports, client, analisys_id):
             LOG.debug("Storing done for report " + str(report_id))
             report_ids.append(report_id)
 
-    def handle_results(self, client=None):
+        return report_ids
+
+    def handle_results(self, client):
         """
         Send the plist content to the database.
         Server API calls should be used in one connection.
```

Comment:
The same _one connection_ thing pointed out by @dkrupp earlier is here. We should clarify this, somehow.